### ISSUE TITLE
Navigating file explorer by keyboard is now broken

### ISSUE THREAD

**Issue by frankreporting** [kepano/obsidian-minimal#353]
### Is the bug present when using the default Obsidian theme?

No

### Minimal theme version

5.3.0

### Describe the bug

Obsidian insider build recently introduced the ability to put keyboard focus on the file explorer so you can navigate the file tree with arrow keys. I think it was in Minimal 5.2 when that was introduced, and it worked perfectly. In the latest update, however, you can no longer see which file has focus as you navigate the file explorer in the sidebar. 

The issue appears to be related to the following style in `src/scss/obsidian/file-nav.scss`:

```css
.workspace-leaf.mod-active .nav-folder.has-focus > .nav-folder-title,
.workspace-leaf.mod-active .nav-file.has-focus {
  border:none;
}
```

By default, the border should be visible so you can see which file or folder you're trying to select.

### Debug info

SYSTEM INFO:
	Obsidian version: v0.15.3
	Installer version: v0.14.6
	Operating system: Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:37 PDT 2022; root:xnu-8020.121.3~4/RELEASE_ARM64_T6000 21.5.0
	Login status: logged in
	Catalyst license: vip
	Insider build toggle: on
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: Minimal
	Snippets enabled: 11
	Safe mode: off
	Plugins installed: 60
	Plugins enabled: 57
		1: Metadata Extractor
		2: MetaEdit
		3: Remember cursor position
		4: ReadItLater
		5: QuickAdd
		6: Quick Switcher++
		7: Quick Explorer
		8: Paste URL into selection
		9: Pane Relief
		10: Pandoc Plugin
		11: Note Folder Autorename
		12: Natural Language Dates
		13: Multi-cursor
		14: Minimal Theme Settings
		15: Markdown Attributes
		16: Obsidian42 - Jump-to-Date
		17: Obsidian42 - BRAT
		18: Readwise Official
		19: Jump to link
		20: Link Favicons
		21: Imgur Plugin
		22: Hotkeys++
		23: Hotkeys for templates
		24: Templater
		25: Typewriter Scroll
		26: Hider
		27: Get Info
		28: CustomJS
		29: Dataview
		30: Advanced Cursors
		31: Advanced Mobile Toolbar
		32: Advanced Slides
		33: Advanced Tables
		34: Advanced URI
		35: Another Quick Switcher
		36: Auto Link Title
		37: Better Word Count
		38: Calendar
		39: Citations
		40: Contextual Typography
		41: Copy as HTML
		42: Customizable Page Header and Title Bar
		43: Editor Syntax Highlight
		44: Find unlinked files and unresolved links
		45: Focus Mode
		46: Folder Note
		47: Force note view mode
		48: Show Whitespace
		49: Sortable
		50: Style Settings
		51: Tag Wrangler
		52: Task Collector (TC)
		53: User Plugins
		54: Wikilinks to MDLinks
		55: Note Auto Creator
		56: Reading Time
		57: Code Editor Shortcuts

RECOMMENDATIONS:
	Custom theme: for cosmetic issues, please first try updating your theme to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Comment by WeberAlex** [comment]
I have just installed the theme 5.03 in Obsidian 1.5.9 (Win) and that's the first thing I noticed (👍 for the bug report above):

[Obsidian Release v0.15.6](https://forum.obsidian.md/t/obsidian-release-v0-15-6/40227)

> Keyboard Accessibility
> Added keyboard navigation to the File Explorer, Tag pane, and Search views Arrow keys to move through items, Enter / Shift+Enter to select.

Keyboard navigation stops working when the Minimal theme is selected.

**Comment by frankreporting** [comment]
So it seems to be strictly a style fix to restore the visual cue when navigating the file tree. I can edit the name of this issue if it helps make that clearer.

**Comment by janpeeters70** [comment]
I experienced the same when installing Minimal 5.3.2 with BRAT plugin. When I installed the release version of 5.3 all was good.

**Comment by kepano** [comment]
This will be fixed in Minimal 6.0 which will coincide with the next major version of Obsidian

**Comment by janpeeters70** [comment]
6.0… a whole number version update, that sounds exciting! :-)
